kinesis_video_streamer/launch/kinesis_video_streamer.launch.py

```diff
@@ -21,19 +21,17 @@
 # Argument names
 NODE_NAME = "node_name"
 CONFIG = "config"
+AWS_REGION = "aws_region"
+STREAM_NAME = "stream_name"
+REKOGNITION_DATA_STREAM = "rekognition_data_stream"
+
 MAX_STRING_LEN = 128
 
-def get_logger_config_path():
+def get_logger_config_path(config_yaml):
```

Comment:
I'd pass in `configured_fallback_path` here rather than all the yaml.

Reply:
Done.